### ISSUE TITLE
Impossible de fermer une discussion à partir du front

### ISSUE THREAD

**Issue by restuccia** [etalab/data.gouv.fr#800]
À partir du très beau frontend de data.gouv.fr ce n'est pas possible de fermer une discussion.

![Capture d’écran 2022-04-01 à 18 18 55](https://user-images.githubusercontent.com/51122275/161303129-55a219ef-dbdf-4fb6-be89-8961713ea0c9.jpg)

C'est possible, en revanche, à partir de l'admin

![Capture d’écran 2022-04-01 à 18 19 05](https://user-images.githubusercontent.com/51122275/161303165-a7976bcf-c8da-4944-9829-e67b31358301.jpg)

**Comment by maudetes** [comment]
Yes, on pourrait éventuellement rajouté ça ! vous en aviez discuté @agarrone ?

**Comment by agarrone** [comment]
Good point, je pense qu'on gagnerait à faire plus d'interactions sur le front. 
On peut le mettre dans la liste des choses à penser lorsqu'on refera les commentaires ?

**Comment by restuccia** [comment]
moi quand je découvre qu'on va refaire les commentaires 👀

**Comment by agarrone** [comment]
A traiter dans l'EPIC commentaire
